### DISCUSSION TITLE
Manual/class-PhysicsManager

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#64] [Notes]
# Manual/class-PhysicsManager

https://docs.unity3d.com/Manual/class-PhysicsManager.html

**Comment by vertxxyz** [comment]
If you disable auto-simulation you can still query the physics world with raycasts. Be aware that colliders will not update with their transforms unless [`Physics.SyncTransforms`](https://docs.unity3d.com/ScriptReference/Physics.SyncTransforms.html) is called, or the legacy Auto Sync Transforms is enabled.
